notebooks/standardize_events_data.ipynb

```diff
@@ -1071,27 +1070,24 @@
     "                'receiver': data_list[2],\n",
     "                'timestamp': data_list[3]\n",
     "            })\n",
-    "        else:\n",
-    "            return pd.Series()\n",
+    "       else:\n",
+    "            raise ValueError(\"Unrecognized key_name\") # Raising an error for unrecognized key_name\n",
     "    except Exception as e:\n",
     "        # Handle any parsing errors\n",
-    "        return pd.Series()\n",
+    "        raise RuntimeError(f\"Failed to extract data from row: {str(e)}\") from e\n",
     "\n",
     "# Filter the dataset for 'hashstack_v1' protocol\n",
     "hashstack_v1_data = df_events[df_events['Protocol'] == 'hashstack_v1']\n",
     "\n",
     "# Apply the extraction function to each row\n",
-    "extracted_elements = hashstack_v1_data.apply(extract_hashstack_v1, axis=1)\n",
+    "standardized_hashstack_v1_events = hashstack_v1_data.apply(standardize_hashstack_v1_event, axis=1)\n",
     "\n",
     "# Concatenate the extracted elements with the original dataframe\n",
-    "standard_hashstack_v1 = pd.concat([hashstack_v1_data[['block_hash', 'block_number', 'transaction_hash', 'timestamp', 'key_name']], extracted_elements], axis=1)\n",
-    "\n",
-    "# Display the resulting dataframe\n",
-    "standard_hashstack_v1.head()\n",
+    "standardized_hashstack_v1_events = pd.concat([hashstack_v1_data[['block_hash', 'block_number', 'transaction_hash', 'timestamp', 'key_name']], standardized_hashstack_v1_events], axis=1)\n",
     "\n",
     "# Uncomment the following below to Save the resulting dataframe to a new CSV file\n",
-    "# output_file_path = 'extracted_hashstack_v1_events.csv'\n",
-    "# standard_hashstack_v1.to_csv(output_file_path, index=False)"
+    "# output_file_path = 'standardized_hashstack_v1_events.csv'\n",
+    "# standardized_hashstack_v1_events.to_csv(output_file_path, index=False)"
    ]
   },
   {
```

Comment:
The code for Nostra Mainnet should be adjusted according to the comments above as well. Moreover, you need to copy the setting for `NOSTRA_MAINNET_TOKENS_TO_TOKEN_TYPES` and `NOSTRA_MAINNET_TOKEN_TYPES_TO_EVENT_TYPES` from `src`.